app/views/items/new.html.erb

```diff
@@ -4,19 +4,19 @@
   <%= simple_form_for @item, url: items_path do |f| %>
     <%= f.error_notification %>
 
-    <%= f.input :title, label: 'Iphone:' %>
+    <%= f.input :title, placeholder: 'Macbook', label:'Give a sexy Title:' %>
 
-    <%= f.input :description, label: 'Useful for Students:' %>
+    <%= f.input :description, placeholder: 'Great for student', label:'Description:' %>
 
-    <%= f.input :condition, label: 'Excellent:' %>
+    <%= f.input :condition, placeholder: 'Excellent', label:'Condition = Excellent, Good, Average, Bad, Broken:' %>
 
-    <%= f.input :year_made, label: '2020:' %>
+    <%= f.input :year_made, placeholder: '2020', label:'Year Made:' %>
 
-    <%= f.input :number_available, label: '1:' %>
+    <%= f.input :number_available, placeholder: '1', label:'Number Available:' %>
```

Comment:
Suggest to add min_max: true, this will work with the validations in the model and start the step from 1

Reply:
added in min_max: true